src/routes/CompanyRoute.js

```diff
@@ -2,14 +2,17 @@ import React from "react";
 import { Route, Redirect } from "react-router-dom";
 
 import { isAuthenticated, isCompany } from "../services/auth";
+import Layout from "../components/layout/Layout";
 
 const CompanyRoute = ({ component: Component, ...rest }) => (
   <Route
     {...rest}
     render={props =>
       isAuthenticated() ? (
         isCompany() ? (
-          <Component {...props} />
+          <Layout isAuthenticated isCompany>
```

Comment:
gosto pessoal, colocaria só `authenticated` e `company` e `admin`

Reply:
vou manter com o `is` só pelo padrão